### DISCUSSION TITLE
SWAMP alternative for lightspeed or updated SWAMP?

### DISCUSSION THREAD

**Discussion by viperbot2000** [3kh0/ext-remover#550] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I used to do swamp every mornin for gogaurdian and lightspeed, but one day it stopped working
https://github.com/SnesFX/swamp-Launcher-for-ChromeOS
swamp ultra never worked... Is there a swamp alternative for lightspeed or gogaurdian? plz help

**Comment by Aka-but-nice** [comment, marked as answer]
Well, no, you have the two blockers, and at this point, there are no exploits yet, your best choice is unenrolling, but I do not recommend doing that, it could get your device prosecuted.

**Reply by viperbot2000** [reply]
ok, let me know when u have one that works, sh1mmer never worked, the IT warned me not to try it. it says external usb detected or something.

**Reply by viperbot2000** [reply]
prosecuted?they destroy them... wow

**Reply by Aka-but-nice** [reply]
Wait the it knew about sh1mmer

**Reply by viperbot2000** [reply]
yea... a couple friends of mine got suspended for that. they told the IT EVERYTHING

**Reply by Aka-but-nice** [reply]
> ok, let me know when u have one that works, sh1mmer never worked, the IT warned me not to try it. it says external usb detected or something.

DM me on discord, I have an exploit.

**Reply by Overreationary** [reply]
Is there any way that's not blocked for you to tell me the exploit? Or is it just him lol?

**Reply by Aka-but-nice** [reply]
> Is there any way that's not blocked for you to tell me the exploit? Or is it just him lol?

No, it works for all but, I don't want it very much public right now, DM me, if you want the exploit, "Aka, but nice#5094" on discord.

**Comment by Overreationary** [comment]
I sent you a friend request.
